.gitlab-ci.yml

```diff
@@ -5,15 +5,21 @@
   tags:
     - docker
 
+clang-format:
+  extends: .in-prplmesh-builder
+  stage: build
+  script:
+    # Run clang-format and check there are no modified files by checking git status
+    - ./clang-format.sh
+    - git update-index -q --refresh
+    - git diff-index -p --exit-code HEAD
```

Comment:
Does this create the patch? If so, what is the patch name? where will it be stored? I think it's worth adding it to the above comment.